Functions.Templates/Templates/SqlInputBinding-JavaScript/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "defaultFunctionName": "SqlInputBindingJavaScript",
```

Comment:
Just for consistency across all templates, do we want the function name to be just 'SqlInputBinding' or also include the language? @Charles-Gagnon @dzsquared

Reply:
Just SqlInputBinding - they (should) never have multiple bindings with different languages in a single project so no need to differentiate the names.